source/vocab/iiif-presentation-annotations.ttl

```diff
@@ -0,0 +1,118 @@
+@prefix iiif_pres3: <http://iiif.io/api/presentation/3#> . # Funkar denna urin?
+@prefix oa: <http://www.w3.org/ns/oa#> . # Funkar denna urin?
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix sdo: <http://schema.org/> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix bf2: <http://id.loc.gov/ontologies/bibframe/> .
+@prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/> .
+@prefix : <https://id.kb.se/vocab/> .
+
+##
+# IIIF PRESENTATION API 3.0
+
+:id owl:equivalentProperty iiif_pres3:id .
+:type owl:equivalentProperty iiif_pres3:type .
```

Comment:
Arent't these terms too generic and overloaded to be defined at the KBV "top level"?